packages/client/lib/execution/execution.ts

```diff
@@ -56,15 +56,21 @@ export abstract class Execution {
    */
   async open(): Promise<void> {
     this.started = true
-    this.config.logger.info('Setup EVM execution.')
+    if (this.config.execution) {
```

Comment:
i think we should not change the log UX here as the execution could still be triggered by beacon sync for e.g. and then stop would not log the stop message.

instead we should just add a log when we trigger execution run in full service start

Reply:
👍

Could you eventually directly push ah update?